core/src/main/scala/com/evolutiongaming/kafka/flow/ConsumerFlow.scala

```diff
@@ -132,4 +104,50 @@ object ConsumerFlow {
 
   }
 
+  private class RebalanceListenerImpl[F[_]: BracketThrowable: LogOf](
```

Comment:
Create a new trait with implementation in companion object, so the unit could be tested in isolation and stubbed easily?

Reply:
Done